queries/cedar/highlights.scm

```diff
@@ -0,0 +1,69 @@
+; Function calls
+(call_expression
+  function: (selector_expression
+    field: (field_identifier) @function.method))
+
+[
+  "contains"
+  "containsAll"
+] @function
+
+; Annotations
+(annotation) @attribute
+
+; Identifiers
+[
+  (entity)
+  (identifier)
+] @variable
+
+[
+  (principal)
+  (action)
```

Comment:
Should these nodes be the anonymous node equivalents? Looks like these highlights don't get applied because sometimes an action is not exposed as `(action)`, but rather as `"action"` in, e.g., `(action_eq_constraint)`

Reply:
That sounds like a consistency issue that should be fixed in the parser.